packages/moonstone/Panels/Header.module.less

```diff
@@ -62,12 +62,12 @@
 		height: @moon-header-height-medium;
 
 		.title {
-			// This `margin-top: -3px;` is intended to account for the height disparity and visual
+			// This `margin-top: 2px;` is intended to account for the height disparity and visual
 			// offset when switching between Header+Input and a Standard Header's title height.
 			// The height is determined by the line-height below, but we want to maintain that as a
 			// relative unit (`em`) rather than dictate specific pixels. The -3px offset may need to
 			// be updated or removed to accomodate taller glyphs.
-			margin-top: -3px;
+			margin-top: 2px;
```

Comment:
This is indeed not ideal. Things that affect layout really shouldn't have a non-multiple of 3. :\